docs/product-getting-started-guide/src/main/asciidoc/task_new-project.adoc

```diff
@@ -1,32 +1,25 @@
 [id='_importing_a_getting_started_business_project']
-= Importing a Getting Started Business Project
+= Creating the traffic violations project
 
-The following chapter instructs you how to import an incomplete business process application. You will extend the application in later chapters.
-
-[IMPORTANT]
-====
-The business process application example includes features that are Technology Preview only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend to use them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
-For more information on Red Hat Technology Preview features support scope, see https://access.redhat.com/support/offerings/techpreview/.
-====
-
-[float]
-== Prerequisites
-
-* Installed Red Hat JBoss EAP 7.0. For further information, see https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.0/html/installation_guide/[_Red Hat JBoss EAP 7.0 Installation Guide_].
-* Installed {PRODUCT}. For further information, see the {INSTALLATION_GUIDE}.
-* {PRODUCT} is running and you can log in to Business Central with the `admin` role. For further information, see the {INSTALLATION_GUIDE}.
+. Log in to {CENTRAL}.
```

Comment:
Procedure modules (and ref modules, etc.) need a context statement that gives this module its place in the world, i.e., its context (go figure). Review the [module templates](https://mojo.redhat.com/docs/DOC-1099486#jive_content_id_Master_source_files_in_asciidoc_format) to see what I mean. Or take a look at my guided decision table or my uploaded decision table assembly for more examples. Not that my work's a prototype at all, just sayin.

Reply:
Added context statement.